exercises/kindergarten-garden/description.md

```diff
@@ -56,3 +56,6 @@ Then if asked for Alice's plants, it should provide:
 While asking for Bob's plants would yield:
 
 - Clover, grass, clover, clover
+
+
+**YOU MAY ALSO NEED TO REPORT ON OTHER CLASSES WITH DIFFERENT STUDENTS.**
```

Comment:
This feels to me like someone shouting orders at me.

Could this perhaps be worded in a more friendly way?

Reply:
Should I reduce the emphasis?

Reply:
I think it should explain how the sub gardens are associated with the other names... And not just tell me that names may be different. 

The description says we have a to m, but now all of a sudden you tell me I need to know the answer for z...

Reply:
Yeah. I was thinking that too!

@Insti Please explain it to me and @NobbZ.

Reply:
@mhdsmr2 you added the sentence that there might pop a set of different names may pop up, so it's your task to explain...

Reply:
I do not think that the less emphased version is any better…

It still leaves many questions open.